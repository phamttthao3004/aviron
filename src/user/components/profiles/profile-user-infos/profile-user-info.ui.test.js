import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ProfileUserInfoUI from './profile-user-info.ui';

describe('Profile Info UI', () => {
    it('Render Profile Info UI component', () => {
        const props = { openDialog: undefined, setOpenDialog: undefined };
        const com = shallow(<ProfileUserInfoUI {...props} />);
        expect(shallowToJson(com)).toMatchSnapshot();
    });
});
