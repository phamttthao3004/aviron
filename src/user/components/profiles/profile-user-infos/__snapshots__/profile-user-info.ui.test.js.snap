// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Info UI Render Profile Info UI component 1`] = `
<div
  className="profile-info"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-2 col-md-offset-1 text-left"
      >
        <div
          className="avatar-box"
        >
          <div
            className="img"
          >
            <img
              alt=""
              src="/images/user/no-avatar.png"
            />
          </div>
        </div>
      </div>
      <div
        className="col-md-9 text-left"
      >
        <div
          className="user-info-box"
        >
          <p>
            Larry Garza Aviron
          </p>
          <p>
            Gabriel Salazar Tomy
          </p>
          <ul>
            <li>
              <img
                alt=""
                src="/images/user/achievement.png"
              />
              <p>
                Professional
              </p>
            </li>
            <li
              className="pointer"
              onClick={[Function]}
              role="presentation"
            >
              <p>
                <i
                  className="icon-users-mm"
                />
                <span>
                  12 friends
                </span>
              </p>
            </li>
          </ul>
        </div>
      </div>
      <CmsDialog
        className=""
        open={false}
      >
        <div
          id="dialog-friend-list"
        >
          <div
            className="friend-list-header"
          >
            <span>
              Friend (12)
            </span>
            <i
              className="icon-close"
              onClick={[Function]}
              role="presentation"
            />
          </div>
          <div
            className="friend-list-content"
          >
            <div
              className="friend-item"
            >
              <div
                className="img"
              >
                <img
                  alt=""
                  src="/images/user/no-avatar.png"
                />
              </div>
              <p
                className="name"
              >
                Gabriel Aviron
              </p>
              <p>
                Gabriel Salazar
              </p>
            </div>
            <div
              className="friend-item"
            >
              <div
                className="img"
              >
                <img
                  alt=""
                  src="/images/user/no-avatar.png"
                />
              </div>
              <p
                className="name"
              >
                Jack Aviron
              </p>
              <p>
                Jack Potter
              </p>
            </div>
            <div
              className="friend-item"
            >
              <div
                className="img"
              >
                <img
                  alt=""
                  src="/images/user/no-avatar.png"
                />
              </div>
              <p
                className="name"
              >
                Sue Aviron
              </p>
              <p>
                Sue Hughes
              </p>
            </div>
            <div
              className="friend-item"
            >
              <div
                className="img"
              >
                <img
                  alt=""
                  src="/images/user/no-avatar.png"
                />
              </div>
              <p
                className="name"
              >
                Gertrude Aviron
              </p>
              <p>
                Gertrude Bowers
              </p>
            </div>
            <div
              className="friend-item"
            >
              <div
                className="img"
              >
                <img
                  alt=""
                  src="/images/user/no-avatar.png"
                />
              </div>
              <p
                className="name"
              >
                Victor Aviron
              </p>
              <p>
                Victor Evans
              </p>
            </div>
            <div
              className="friend-item"
            >
              <div
                className="img"
              >
                <img
                  alt=""
                  src="/images/user/no-avatar.png"
                />
              </div>
              <p
                className="name"
              >
                Sue Aviron
              </p>
              <p>
                Sue Hughes
              </p>
            </div>
          </div>
        </div>
      </CmsDialog>
    </div>
  </div>
</div>
`;
